yiisoft/yii2-bootstrap/CVE-2015-3397.yaml

```diff
@@ -0,0 +1,8 @@
+title:     JSON Data encoded for use in HTML was not safe to use in IE6/IE7, possible XSS attacks
+link:      http://www.yiiframework.com/news/86/yii-2-0-4-is-released/
+cve:       CVE-2015-3397
+branches:
+    2.0.x:
```

Comment:
does this need to refer to a real branch or is it just informational style? Docs are not really clear about it.